src/layout/SimpleTable/config.ts

```diff
@@ -85,4 +98,10 @@ export const Config = new CG.component({
   )
   .addProperty(
     new CG.prop('size', new CG.enum('sm', 'md', 'lg').setTitle('Size').setDescription('Size of table.').optional()),
+  )
+  .addProperty(
+    new CG.prop(
+      'externalApi',
+      new CG.obj(new CG.prop('id', new CG.str()), new CG.prop('path', new CG.str())).optional().exportAs('DataConfig'),
+    ),
```

Comment:
For meg så hadde det kanskje gitt mer mening om man hadde hatt én prop for "kilden" til dataen? Typ: 
```ts
type DataSource = {
  type: "datamodel",
  ...
} | {
  type: "externalApi",
  ...
}
```

Er det mulig å få til? 😊 tror det er lett å forstå for utviklerne (og oss!), i tillegg til at det er veldig lett å utvide senere om vi plutselig trenger det.

Reply:
Tenker også at det hadde vært supernice om man kunne henta dataen basert på dette og så rendra samme "dumme" table for begge use casene, bare med forskjellige data og config. Hva tenker du om det? Nå har jeg ikke satt meg veldig godt inn i det, men er det oppnåelig? 😄

Reply:
> For meg så hadde det kanskje gitt mer mening om man hadde hatt én prop for "kilden" til dataen? Typ:
> 
> ```ts
> type DataSource = {
>   type: "datamodel",
>   ...
> } | {
>   type: "externalApi",
>   ...
> }
> ```
> 
> Er det mulig å få til? 😊 tror det er lett å forstå for utviklerne (og oss!), i tillegg til at det er veldig lett å utvide senere om vi plutselig trenger det.

Usikker på om det er mulig med hvordan kode generering funker, men skal ta en titt :)

Reply:
> Tenker også at det hadde vært supernice om man kunne henta dataen basert på dette og så rendra samme "dumme" table for begge use casene, bare med forskjellige data og config. Hva tenker du om det? Nå har jeg ikke satt meg veldig godt inn i det, men er det oppnåelig? 😄

Enig, er bare en del forskjeller på hva som er støttet i de to så derfor jeg gjorde det sånn for nå. Du kan jo blant annet ikke redigere og slette når du viser ekstern data.

Reply:
> > For meg så hadde det kanskje gitt mer mening om man hadde hatt én prop for "kilden" til dataen? Typ:
> > ```ts
> > type DataSource = {
> >   type: "datamodel",
> >   ...
> > } | {
> >   type: "externalApi",
> >   ...
> > }
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Er det mulig å få til? 😊 tror det er lett å forstå for utviklerne (og oss!), i tillegg til at det er veldig lett å utvide senere om vi plutselig trenger det.
> 
> Usikker på om det er mulig med hvordan kode generering funker, men skal ta en titt :)

Hørte med Ole Martin her, virker ikke som dette er mulig uten større forandringer.

Har lagt til en datamodell validering som gir deg en error om du har lagt til begge, tenker det er greit?

Reply:
Ja, det bør holde 😄